packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -141,29 +172,51 @@ export default function LabeledField(props: Props) {
     }
 
     function maybeRenderError(): React.ReactNode | null | undefined {
-        if (!error) {
-            return null;
-        }
-
         return (
             <React.Fragment>
                 <Strut size={spacing.small_12} />
-                <LabelSmall
-                    style={light ? styles.lightError : styles.error}
-                    role="alert"
-                    id={errorId}
-                    testId={testId && `${testId}-error`}
+                <View
+                    style={styles.errorSection}
+                    id={errorSectionId}
+                    testId={testId && `${testId}-error-section`}
+                    aria-live="assertive"
                 >
-                    {error}
-                </LabelSmall>
+                    {error && (
+                        <>
+                            <PhosphorIcon
+                                icon={WarningCircle}
+                                style={[
+                                    styles.errorIcon,
+                                    light ? styles.lightError : styles.error,
+                                ]}
+                                role="img"
+                                aria-label={labels.errorIconAriaLabel}
+                            />
+                            <LabelSmall
+                                style={[
+                                    styles.textWordBreak,
+                                    styles.errorMessage,
+                                    light ? styles.lightError : styles.error,
+                                ]}
+                                id={errorId}
+                                testId={testId && `${testId}-error`}
```

Comment:
Related to my previous comment about simplifying the error section + testId.

```suggestion
                                testId={testId && `${testId}-error`}
```